gfx/drivers_context/drm_ctx.c

```diff
@@ -563,7 +563,8 @@ static bool gfx_ctx_drm_egl_set_video_mode(gfx_ctx_drm_data_t *drm)
       case GFX_CTX_OPENGL_ES_API:
       case GFX_CTX_OPENVG_API:
 #ifdef HAVE_EGL
-         if (!egl_init_context(&drm->egl, (EGLNativeDisplayType)g_gbm_dev, &major,
+         if (!egl_init_context(&drm->egl, EGL_PLATFORM_GBM_KHR,
```

Comment:
Just a side note: linux systems with old EGL headers will fail on this (ubuntu 14.04 for example), there's no EGL_PLATFORM_GBM_KHR in < EGL/eglext.h >

libegl1-mesa 10.1.3

```
gfx/drivers_context/drm_ctx.c: In function ‘gfx_ctx_drm_egl_set_video_mode’:
gfx/drivers_context/drm_ctx.c:566:43: error: ‘EGL_PLATFORM_GBM_KHR’ undeclared (first use in this function)
          if (!egl_init_context(&drm->egl, EGL_PLATFORM_GBM_KHR,
                                           ^
```

Need to build with --disable-wayland --disable-egl